yellowbrick/classifier/classification_report.py

```diff
@@ -134,6 +145,11 @@ def draw(self):
                 value = cr_display[x,y]
                 svalue = "{:0.3f}".format(value)
 
+                # change the svalue for support (when y == 3) because we want
+                # to label it as the actual support value, not the percentage
```

Comment:
After seeing some of the example images, I'd actually recommend that we make this an option for the user in `__init__` - I'm thinking the user should specify:

1. Show support (or don't show support for backward compatibility) 
2. Show support as percent 
3. Show support as real number 

We can do this with a single argument:

`support="count"` 

Where support can be None, False, True, or one of "percent", "count". If None or False, do not display support; if "count" displays the actual count, otherwise display percent. 

If not in this PR, we could open a high priority issue for this.